xclim/core/units.py

```diff
@@ -591,7 +594,7 @@ def to_agg_units(
             out.attrs["units"] = pint2cfunits(orig_u * freq_u)
     else:
         raise ValueError(
-            f"Aggregation op {op} not in [min, max, mean, std, doymin, doymax, count, integral]."
+            f"Unknown aggregation op {op}. Known ops are [min, max, mean, std, var, doymin, doymax, count, integral, sum]."
```

Comment:
Missing `amax` and `amin`

Reply:
@coxipi  what are "amin" and "amax" ?

Reply:
https://numpy.org/doc/stable/reference/generated/numpy.amin.html
min/max of an array in numpy. 

When changing `to_agg_units` in https://github.com/Ouranosinc/xclim/pull/1443/files , I noticed that some functions use `amin/amax` instead of `min/max`, not including `amin/amax` caused some errors in the unit tests. The documentation in `to_agg_units` does not reflect this, only mentioning `min/max`, maybe  it would be better to also mention those functions. Would it be useful if I check which functions use `amin/amax`?